### ISSUE TITLE
Organic Chemistry - LaTeX coding of double bonds and triple bonds in Exercises Tool 

### ISSUE THREAD

**Issue by ww-openstax** [openstax/cnx-recipes#5136]
# Brief Description of Scenario
**In Text:** we are coding double and triple bonds in MathML and including the "entire element" in the MathML
![image](https://github.com/openstax/cnx-recipes/assets/89537088/92836517-d9f8-44fe-bfe1-0e78bba99e70)

**In Exercises Tool:** we are coding double and triple bonds in LaTeX and including only the "bond" in the LaTeX
![image](https://github.com/openstax/cnx-recipes/assets/89537088/5b0d0328-30bc-4157-8331-00eb2c2287e3)

Which gives the following output:
<img width="609" alt="image" src="https://github.com/openstax/cnx-recipes/assets/89537088/635d886e-8710-498f-93b8-c018624c7651">

**QUESTION:** **Is the LaTeX coding in the screenshot above correct and acceptable for double bonds and triple bonds in the Exercises?** 
Note: this is the coding we have been using since day one and has been the coding used for the creation of all reviewed PDFs

# Pattern of Occurrence 

# Acceptance Criteria

  
# Sample Mark-Up
```
<!-- Please paste here if applicable. -->
```

# Location(s)
- **osbooks repo:** osbooks-organic-chemistry
- **collection slug:** organic-chemistry
- **ref/branch:**
- **chapter/section:** 
- **surrounding text:**

**Comment by ww-openstax** [comment]
The WiseWire team has been following the LaTeX style OpenStax used in Exercises in the original Chapter 01. And we want to confirm that this is what is expected. See Nickname OrgChem_C01_Q010 that OpenStax created.

**Comment by openstaxalina** [comment]
Yes, @lwtchu and I both reviewed this and this is correct and acceptable. Closing!
